Polyhedron/demo/Polyhedron/Plugins/Mesh_3/raw_image.ui

```diff
@@ -1,62 +1,81 @@
+  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
-    <widget class="QGroupBox" name="groupBox_2" >
-     <property name="title" >
+    <widget class="QGroupBox" name="groupBox_2">
+     <property name="title">
       <string>Image &amp;value type</string>
      </property>
-     <layout class="QGridLayout" name="gridLayout" >
-      <item row="2" column="0" >
-       <widget class="QRadioButton" name="short_bt" >
-        <property name="text" >
+     <layout class="QGridLayout" name="gridLayout">
+      <item row="2" column="0">
+       <widget class="QRadioButton" name="short_bt">
+        <property name="toolTip">
+         <string>Word kind of the image : short int.</string>
+        </property>
+        <property name="text">
          <string>Short (16 bits)</string>
         </property>
        </widget>
       </item>
-      <item row="3" column="0" >
-       <widget class="QRadioButton" name="float_bt" >
-        <property name="text" >
+      <item row="3" column="0">
+       <widget class="QRadioButton" name="float_bt">
+        <property name="toolTip">
+         <string>Word kind of the image : float.</string>
+        </property>
+        <property name="text">
          <string>Float</string>
         </property>
        </widget>
       </item>
-      <item row="0" column="1" >
-       <widget class="QRadioButton" name="int_bt" >
-        <property name="text" >
+      <item row="0" column="1">
+       <widget class="QRadioButton" name="int_bt">
+        <property name="toolTip">
+         <string>Word kind of the image : integer.</string>
+        </property>
+        <property name="text">
          <string>Int (32 bits)</string>
         </property>
        </widget>
       </item>
-      <item row="3" column="1" >
-       <widget class="QRadioButton" name="double_bt" >
-        <property name="text" >
+      <item row="3" column="1">
+       <widget class="QRadioButton" name="double_bt">
+        <property name="toolTip">
+         <string>Word kind of the image : double.</string>
+        </property>
+        <property name="text">
          <string>Double</string>
         </property>
        </widget>
       </item>
-      <item row="2" column="1" >
-       <widget class="QCheckBox" name="signed_bt" >
-        <property name="text" >
+      <item row="2" column="1">
+       <widget class="QCheckBox" name="signed_bt">
+        <property name="toolTip">
+         <string>Specifies if the image's word kind is signed.</string>
+        </property>
+        <property name="text">
          <string>Signed</string>
         </property>
        </widget>
       </item>
-      <item row="0" column="0" >
-       <widget class="QRadioButton" name="char_bt" >
-        <property name="text" >
+      <item row="0" column="0">
+       <widget class="QRadioButton" name="char_bt">
+        <property name="toolTip">
+         <string>Word kind of the image : unsigned byte.</string>
```

Comment:
The tooltip's text is wrong: it can also be a "*signed* char". Anyway, just remove all those tooltips!